html/semantics/selectors/case-sensitivity-of-selectors/element-selectors.html

```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<meta charset=utf8>
+<title>CSS element selector case-sensitivity</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
```

Comment:
Could you add a `<link rel=help>` with the relevant spec-link?

Reply:
if you don't know, for HTML you can always take the last path segment as a fragment (i.e., #case-sensitivity-of-selectors).